### DISCUSSION TITLE
Codes for subcategories

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1493] [Q&A] [answered]
I have created codes for subcategories (Ukraine SDG, indicator 3.7.1.
Additionally, I added the text for categories in the data.yml in the translation/en and translation/uk folders 

However, I have only codes for categories on the site.

Additionally, for the indicators, which were ok, the translations also does not work now - the site shoes codes instead of text (for example, 2.2.1)

Could you, please, help to fix it

**Comment by brockfanning** [comment, marked as answer]
@ukraine-sdg Please see my answer at #1491 - I believe that your translations files (data.yml) could not be parsed because of some tab characters.
